README.md

```diff
@@ -13,19 +13,18 @@ git cms-init
 git remote add btv-cmssw https://github.com/cms-btv-pog/cmssw.git
 
 #Add DeepFlavour and its dependencies
-git cms-merge-topic capalmer85:btagSFupdatesForTTbar
-git cms-merge-topic cms-btv-pog:DeepFlavourUpdates-from-CMSSW_9_4_4
+git cms-merge-topic cms-btv-pog:DeepFlavourNewTraining-from-CMSSW_10_1_2_patch2
 
-git clone -b 9_4_X_v1.06 --depth 1 https://github.com/cms-btv-pog/RecoBTag-PerformanceMeasurements.git RecoBTag/PerformanceMeasurements
+git clone -b https://github.com/cms-btv-pog/RecoBTag-PerformanceMeasurements.git RecoBTag/PerformanceMeasurements
```

Comment:
This should be 
git clone -b 10_1_X --depth 1 blah blah
-b clones and checks out the 10_1_X branch. depth 1 slims the commit history to be manageable for the end user, you should not do it :)